src/intentManager.js

```diff
@@ -219,8 +259,17 @@ define([
         getMugTypes: function () {
             var types = this.__callOld();
             types.normal.AndroidIntent = AndroidIntent;
+            types.normal.PrintIntent = PrintIntent;
             return types;
         },
+        getAdvancedQuestions: function () {
+            var ret = this.__callOld();
+            ret.push("AndroidIntent");
+            if (this.opts().features.printing) {
+                ret.push("PrintIntent");
```

Comment:
What's this? Another question type in the Advanced menu?! Seems like this might be a reasonable thing to add to this list of presets being added for the area mapper spec.